src/ParameterTypes/passive_ren.jl

```diff
@@ -51,14 +52,19 @@ function PassiveRENParams{T}(
         error("Input and output must have the same dimension for passiveREN")
     end
 
+    # Check ρ and ν
```

Comment:
@MrstupidJ could you please clarify this error message? When you say that `ρ and ν cannot be all positive`, which of the following should be satisfied?

- $\rho \le 0$ and $\nu \le 0$
- $\rho \ge 0$ and $\nu \ge 0$
- $\rho \nu \le 0$

At the moment it's a little unclear to me, but maybe that's because I'm not as familiar with passivity.

Reply:
In theory, $\rho \le 0$ or $\nu \le 0$ indicates some shortage of passivity, i.e. the model or system is not passive. In the document of [PassiveRENParams](https://github.com/acfr/RobustNeuralNetworks.jl/blob/3bc28d5f41b31be31f4991e1418634f5e25d067d/src/ParameterTypes/passive_ren.jl#L16) we should have suggested that $\rho$ and $\nu$ are both non-negative values. 

We can have both $\rho \ge 0$ and $\nu \ge 0$, but this will introduce some problems when we parameterize [this matrix](https://github.com/acfr/RobustNeuralNetworks.jl/blob/3bc28d5f41b31be31f4991e1418634f5e25d067d/src/ParameterTypes/general_ren.jl#L88), ending up with a strange condition ($\nu \le \frac{1}{4\rho}$) that does not make sense in either theory or application. To avoid this confusion, I added this error message to make sure only one of them is positive.

Reply:
I put it [here](https://github.com/acfr/RobustNeuralNetworks.jl/blob/42a22b1e86e5c80b42ab485f82cde5ec7dc2da80/src/ParameterTypes/passive_ren.jl#L26), but also left a "back door" if one needs to set negative $\nu$ or $\rho$ if necessary.

Reply:
Your comment doesn't say anything about needing at least one of $\rho$ or $\nu$ to be zero though? I just want to make sure that there isn't any undefined behaviour. Eg: someone might accidentally set $\nu,\rho = -0.5$ and no errors would pop up (or if they do they might not be very descriptive).